### PR TITLE
chore(deps): bump `@metamask/keyring-api` to v0.2.1

### PR DESCRIPTION
**Pull request by gantunesr**
Resolves: https://github.com/MetaMask/accounts-planning/issues/43

### CODE REVIEW

File: src/SnapKeyring.test.ts [hunk 1]
```diff
@@ -1,5 +1,9 @@
 import { TransactionFactory } from '@ethereumjs/tx';
-import { KeyringAccount } from '@metamask/keyring-api';
+import {
+  EthMethod,
+  EthAccountType,
+  InternalAccount,
+} from '@metamask/keyring-api';
```
Comment (gantunesr) [inline src/SnapKeyring.test.ts:6]: The tests regarding `serialize` and `deserialize` are not working currently because of the data type change.

Comment (gantunesr) [reply src/SnapKeyring.test.ts:6]: @danroc @montelaidev regarding our conversation earlier today, when should the `metadata` attribute be added to the accounts object to comply with the `InternalAccount`? And what's the source of data for the fields `snap` and `name`?

Comment (danroc) [reply src/SnapKeyring.test.ts:6]: `snap` metadata is filled by the `SnapKeyring` (this repo), the `name` should be filled by the `AccountsController`.

**Comment by danroc** [comment]
Merged into #75.

**Pull request closed without merging**